src/components/event.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+import { GalleryEvent } from '../types';
+
+export interface IGalleryEventComponentProps {
+  event: GalleryEvent;
+}
+
+export class GalleryEventComponent extends React.Component<IGalleryEventComponentProps, {}> {
+  public render() {
+    return <div>
+      <a className="db center mw5 black link dim"
+         href="{event.website || event.gallery.website}">
```

Comment:
Should be `href={event.website || event.gallery.website}>` I think

Reply:
(as in no quotes) around the curly brackets

Reply:
Yeah I noticed that >.<

Fixed in https://github.com/tlwr/some-gallery-thing/commit/8a5cf16bb990081b0ca7d0a0be55bc5174ea6b67